modules/storage/storage.go

```diff
@@ -167,22 +172,17 @@ func (s *storageModule) initialize(policy modules.StoragePolicy) error {
 
 	// make sure new pools are mounted
 	log.Info().Msgf("Making sure new volumes are mounted")
-	for _, pool := range newPools {
-		if _, mounted := pool.Mounted(); !mounted {
-			log.Debug().Msgf("Mounting volume %s", pool.Name())
-			if _, err = pool.Mount(); err != nil {
+	for idx := range newPools {
```

Comment:
Not really _too_ important here since its just a string, but yes. Could have left it here, but I already changed it and it IS a very small performance benefit